tests/xgboost/test_xgboost_model_export.py

```diff
@@ -212,6 +213,58 @@ def test_model_save_persists_requirements_in_mlflow_model_directory(
     _compare_conda_env_requirements(xgb_custom_env, saved_pip_req_path)
 
 
+@pytest.mark.large
+def test_save_model_with_pip_requirements(xgb_model, tmpdir):
```

Comment:
Can we add test coverage for log_model as well?

Reply:
Absolutely!